pkg/config/setup/config.go

```diff
@@ -1534,6 +1534,13 @@ func logsagent(config pkgconfigmodel.Setup) {
 	// more disk I/O at the wildcard log paths
 	config.BindEnvAndSetDefault("logs_config.file_wildcard_selection_mode", "by_name")
 
+	// Max size in MB an integration logs file can use
+	config.BindEnvAndSetDefault("logs_config.integrations_logs_files_max_size", 10)
+	// Max disk usage in MB all integrations logs files are allowed to use in total
+	config.BindEnvAndSetDefault("logs_config.integrations_logs_total_usage", 1000)
```

Comment:
1000 MB seems like a lot. 
Maybe we shoul use seomthing smaller by default? 50 or 100MB?

Also we should probaly clarify exactly how this setting interacts with `integrations_logs_disk_ratio` because right now it's not clear without reading the code.

Reply:
Clarified in the comments how `logs_config.integrations_logs_disk_ratio` and `logs_config.integrations_logs_total_usage` interact (the agent calculates the usable space from disk ratio, and then chooses the lowest).

Reply:
Changed the default to 100